lra/coordinator/server/src/main/java/io/helidon/lra/coordinator/LraDatabasePersistentRegistry.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.helidon.lra.coordinator;
+
+import java.net.URI;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import io.helidon.common.reactive.Multi;
+import io.helidon.common.reactive.Single;
+import io.helidon.config.Config;
+import io.helidon.dbclient.DbClient;
+import io.helidon.dbclient.DbTransaction;
+
+import org.eclipse.microprofile.lra.annotation.LRAStatus;
+
+class LraDatabasePersistentRegistry implements LraPersistentRegistry {
+
+    private final Map<String, Lra> lraMap = Collections.synchronizedMap(new HashMap<>());
+    private final Config config;
+    private final DbClient dbClient;
+
+    LraDatabasePersistentRegistry(Config config) {
+        this.config = config;
+        dbClient = DbClient.builder()
+                .config(config.get("db"))
+                .build();
+
+        dbClient.inTransaction(tx -> Multi.concat(
+                                tx.namedDml("create-lra-table"),
+                                tx.namedDml("create-participant-table"))
+                        .ignoreElements())
+                .await();
+    }
+
+    public Lra get(String lraId) {
```

Comment:
class is not public, methods do not need to be public.
If methods implement interface methods, they should be annotated with `@Override`

Reply:
fixed